dashboards-observability/public/components/visualizations/charts/maps/heatmap.tsx

```diff
@@ -58,16 +66,23 @@ export const HeatMap = ({ visualizations, layout, config }: any) => {
     isEmpty(xaxisField) ||
     isEmpty(yaxisField) ||
     isEmpty(zMetrics) ||
-    isEmpty(queriedVizData[xaxisField.label]) ||
+    isEmpty(queriedVizData[xaxisField.name]) ||
```

Comment:
Question: why are we changing from label to name?

Reply:
as we, included time_field(span) into dimensions and id does not contain label as common. in both dimensions and time_field(span)  the common filed was **name**, so we did changed to it.